docs/General/Manuals/client-manual.rst

```diff
@@ -195,18 +195,28 @@ text file if needed.
 Server chat/command line
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The :guilabel:`server chat/command line` is a text box below the server output window. From here you can
+manually enter ``/set`` commands to the server directly if you know what you want to set.
```

Comment:
In short turn multiplayer games, this is used to discuss the settings and other stuff (it's in fact the same box as the in-game chat).